src/main/java/com/kodilla/ecommercee/domain/user/Status.java

```diff
@@ -0,0 +1,48 @@
+package com.kodilla.ecommercee.domain.user;
+
+import com.kodilla.ecommercee.domain.order.Item;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity
+@Table(name = "STATUS;")
```

Comment:
w sumie to bez sensu żeby status był osobną tabelą. to jest właściwość użytkownika

Reply:
Idea jest taka by dane były bardziej spójne.

Reply:
Co ma spójność danych do wydzielenia osobnej tabeli ze statusem?